src/DeltaLake/Extensions/MarshalExtensions.cs

```diff
@@ -1,95 +1,95 @@
-// -----------------------------------------------------------------------------
```

Comment:
Some unintended CRLF change got pushed up in this and a few other files below?

(Not seeing a diff)